x/posts/client/rest/rest.go

```diff
@@ -1,174 +1,42 @@
 package rest
 
 import (
-	"fmt"
-	"net/http"
-
 	"github.com/cosmos/cosmos-sdk/client/context"
-	"github.com/desmos-labs/desmos/x/posts/internal/types"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/rest"
-	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
-
 	"github.com/gorilla/mux"
 )
 
+// REST Variable names
+// nolint
+const (
+	RestCreator      = "creator"
+	RestParentID     = "parent-id"
```

Comment:
May keep query path with `_` but not `-`.

Reply:
Why is this necessary?

```suggestion
	RestParentID     = "parent_id"
```

Reply:
It's a more acceptable convention for query string parameters to have `_`. We keep the service resource names in `-`.

https://specs.openstack.org/openstack/api-wg/guidelines/naming.html#fields-in-an-api-request-or-response-body

Reply:
Done